.golangci.yaml

```diff
@@ -140,6 +132,13 @@ linters:
     - wrapcheck
     # deprecated
     # golangci-lint linters | awk 'f;/Disabled by your/{f=1}' | cut -d":" -f1 | grep deprecated | cut -d" " -f1 | sort
+    - deadcode
     - exhaustivestruct
+    - golint
+    - ifshort
     - interfacer
+    - maligned
     - nosnakecase
+    - scopelint
+    - structcheck
+    - varcheck
```

Comment:
Why keep deprecated list instead of deleting them?

Reply:
Just because we had enabled them at some point and I don't want the information to be lost. We can certainly delete them later once we agree on it.

Reply:
This change actually disable more linters, right? the diff is not clear because the lists are so long, and the commit message you don't list the disabled linters.

I support disabling the deprecated linters but make it clear in the commit message.